cmd/zed/manage/lakemanage/branch.go

```diff
@@ -74,26 +50,25 @@ type compactTask struct {
 	log *zap.Logger
```

Comment:
The task logger should have an extra field that has the name of the task (e.g. "compact" or "vector") but since we only got one I'll do this for the time being.